sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
```

Comment:
I don't think it's a common term. I did coin it, but I'm almost certainly not the first person to do so. I've been using it for a few years for bio gold standards (particularly in gene function prediction), but bronze works for me too :)